packages/filesystem/src/browser/filesystem-preferences.ts

```diff
@@ -26,22 +32,11 @@ export const filesystemPreferenceSchema: PreferenceSchema = {
     }
 };
 
-export interface FileSystemConfiguration {
```

Comment:
Point of proxies to provide type-safe access to preferences, if you remove configuration interfaces and return generic proxies then there is no point in having proxies. You should only replace default configuration by default values form schema.

Reply:
Ok so still keeping the default configuration interface, I tried removing it because I thought it'd be easier for an extension-developer to simply provide a schema with default values, but I see what you meant by using the schema for default preferences in the config, I'll change it.